packages/graphback-datasync/tests/__snapshots__/DataSyncPluginTest.ts.snap

```diff
@@ -103,14 +105,14 @@ input MutateCommentInput {
   title: String
   description: String
   noteId: ID
-  updatedAt: String!
+  _version: Int!
 }
 
 input MutateNoteInput {
   id: ID!
   title: String
   description: String
-  updatedAt: String!
+  _version: Int!
```

Comment:
I would imagine this should not be added to the mutation input - the user should not be able to set the value.

Reply:
Correct me if I am wrong @ssd71 @wtrocki

Reply:
User needs to pass the value unchanged

Reply:
The `CreateXXXInput`, but I think it should be there for the update input, since it will be used for conflicts detection https://github.com/aerogear/graphback/pull/1771/files#diff-be2392f5aaf3d58e02d7e9c6429f279cR146.